src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -1234,7 +1234,7 @@ public void close() throws SecurityException {}
                 logger.addHandler(handler);
                 loggerReferences.add(logger);
             });
-            DeltaSupportLogFormatter.start = start; // match clock time on master
+            DeltaSupportLogFormatter.start = start; // match clock time on Controller
```

Comment:
Should not be upper case.  Not a proper noun.
```suggestion
            DeltaSupportLogFormatter.start = start; // match clock time on controller
```